src/less/skeleton-loader/skeleton-loader.less

```diff
@@ -0,0 +1,252 @@
+@import "../variables/variables.less";
+
+@skeleton-text-border-radius: 3px;
+@skeleton-text-height-small: 16px;
+@skeleton-text-height-large: 24px;
+
+:root {
+    --skeleton-circle-size: 30px;
+    --skeleton-default-height: 40px;
+    --skeleton-rectangular-height: var(--skeleton-default-height);
+    --skeleton-rectangular-width: 120px;
+    --skeleton-square-size: var(--skeleton-default-height);
```

Comment:
Could you condense all of these to just `--skeleton-width: 100%` and `--skeleton-height: 40px`? Then each of the specific components could just set the values directly. Maybe you could also include `--skeleton-radius: 0`